itou/employee_record/admin.py

```diff
@@ -31,8 +32,19 @@ class EmployeeRecordAdmin(admin.ModelAdmin):
     def update_employee_record_as_ready(self, _request, queryset):
         queryset.update(status=Status.READY)
 
+    @admin.action(description="Planifier une notifications de changement 'PASS IAE' pour ces fiches salarié")
+    def schedule_approval_update_notification(self, request, queryset):
+        for employee_record in queryset:
+            models.EmployeeRecordUpdateNotification.objects.update_or_create(
+                employee_record=employee_record,
+                notification_type=models.NotificationType.APPROVAL,
+                defaults={"updated_at": timezone.now},
+            )
+        messages.add_message(request, messages.SUCCESS, f"{queryset.count()} notification(s) planifiée(s)")
```

Comment:
On peut facilement pluralize. Aussi, on peut éviter `.count()`. Même si comme le QS a déjà été évalué et que son cache sera utilisé, l’appel à `len()` est plus clair et éviter une seconde requête SQL par ex. si le message était construit avant l’itération.
```suggestion
        s = pluralizefr(queryset)
        messages.add_message(request, messages.SUCCESS, f"{len(queryset)} notification{s} planifiée{s}")
```